core/src/integration-test/java/com/scalar/db/api/DistributedTransactionAdminIntegrationTestBase.java

```diff
@@ -69,24 +71,29 @@ public abstract class DistributedTransactionAdminIntegrationTestBase {
           .build();
   private TransactionFactory transactionFactory;
   private DistributedTransactionAdmin admin;
+  protected DistributedTransactionAdmin adminWithDebug;
   private String namespace1;
   private String namespace2;
   private String namespace3;
 
   @BeforeAll
   public void beforeAll() throws Exception {
     initialize();
-    transactionFactory = TransactionFactory.create(TestUtils.addSuffix(gerProperties(), TEST_NAME));
+    transactionFactory = TransactionFactory.create(TestUtils.addSuffix(getProperties(), TEST_NAME));
     admin = transactionFactory.getTransactionAdmin();
     namespace1 = getNamespace1();
     namespace2 = getNamespace2();
     namespace3 = getNamespace3();
     createTables();
   }
 
-  protected void initialize() throws Exception {}
+  protected void initialize() throws Exception {
+    Properties debugProperties = TestUtils.addSuffix(getProperties(), TEST_NAME);
+    debugProperties.setProperty(DatabaseConfig.DEBUG, "true");
+    adminWithDebug = TransactionFactory.create(debugProperties).getTransactionAdmin();
```

Comment:
First, thank you for all your suggestions.

Sorry, it's a bit confusing why the `adminWithDebug` initialization is in the `initialize` method only for this class. 
I put it here because in `DistributedTransactionAdminServiceIntegrationTest.initialize` the `adminWithDebug` instance needs to be configured specifically to target an instance of Scalar DB server with debug mode turned on. Since I am also creating a second server instance, I thought it was more appropriate to put it in `initialize` instead of having to overwrite the `beforeAll` method.
https://github.com/scalar-labs/scalardb/blob/849a58e2e90490d0e5080f182dd40d3848ee3b13/server/src/integration-test/java/com/scalar/db/server/DistributedTransactionAdminServiceIntegrationTest.java#L26-L40